botocore/session.py

```diff
@@ -726,12 +721,6 @@ def create_client(self, service_name, region_name=None, api_version=None,
         :param region_name: The name of the region associated with the client.
             A client is associated with a single region.
 
-        :type api_version: string
```

Comment:
Don't think we need to do it in this PR, but it would be nice to get a follow up PR that removes the models of the old api versions in order to reduce botocore's footprint.

Reply:
Agreed.  I can do a separate PR after this one to remove old model data.